picard/tagger.py

```diff
@@ -605,12 +617,28 @@ def handle_command_save_modified(self, argstring):
     def handle_command_scan(self, argstring):
         self.analyze(self.unclustered_files.files)
 
+    def handle_command_set_saved_logs_verbosity(self, argstring):
+        if argstring.upper() in {"DEBUG", "D"}:
+            self.saved_logs_verbosity = logging.DEBUG
+        elif argstring.upper() in {"INFO", "I"}:
+            self.saved_logs_verbosity = logging.INFO
+        elif argstring.upper() in {"WARNING", "W"}:
+            self.saved_logs_verbosity = logging.WARNING
+        elif argstring.upper() in {"ERROR", "E"}:
+            self.saved_logs_verbosity = logging.ERROR
+
     def handle_command_show(self, argstring):
         self.bring_tagger_front()
 
     def handle_command_submit_fingerprints(self, argstring):
         self.acoustidmanager.submit()
 
+    def handle_command_write_logs(self, argstring):
+        with open(argstring, 'w') as f:
```

Comment:
We definitively need to catch exceptions here, for example, if the target file cannot be written, a PermissionError might be raised, leading to a crash of the application.
We should catch all exceptions, and log them to avoid any issue.

Reply:
Just `except Exception as e` and then log it?

Reply:
Yes